Cars/CarsRepository/Interface/ICarRepository.cs

```diff
@@ -0,0 +1,22 @@
+﻿using CarsData.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CarsRepository.Interface
+{
+    public interface ICarRepository
+    {
+        IEnumerable<Car> GetAllCars();
+
+        IEnumerable<Car> GetFavoriteCars { get; }
+
+        IEnumerable<Car> GetCarsByCategory(int categoryId);
+        Car GetObjectCar(int CarId);
```

Comment:
Check all namings to describe method by its work. Figure out what does this method doo and name it right, for example GetCarById, GetAllCars, GetCategories and ect.